src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseByIdTest.java

```diff
@@ -53,8 +55,8 @@ void setUp() {
             DateConverterUtil.toLocalDateTime(SOME_DATE_TIME)
         );
         CourtCaseEntity courtCase = hearingEntity.getCourtCase();
-        courtCase.addProsecutor("aProsecutor");
-        courtCase.addDefendant("aDefendant");
+        courtCase.addProsecutor(someMinimalProsecutorForCase(courtCase));
```

Comment:
this seems quite messy, the fact that we are passing the courtcase into the method, but then add the prosecutor to the courtCase, so its been liked twice.
I thought this normally led to issues in tests.

Reply:
This replaces a method that was added specifically for test in production code which is one of things that needs to be removed.  What issues in test?

Reply:
I can;t remember it was last year, but i think there was persistence problems because the 2 objects were being linked twice.
If someMinimalProsecutorForCase() links the Prosecutor to the case, why do we need `courtCase.addProsecutor` also? Can we call  someMinimalProsecutorForCase by itself?

Reply:
Its a bidirectional relationship so it is by nature linked both ways.  The build is passing so there doesn't seem to be any issues.

No because when we call save(case) we want the save method to iterate through its fields and save the ones that are relevant, in this case save(prosecuter)